libs/blocks/featured-article/featured-article.js

```diff
@@ -1,6 +1,16 @@
 import { getMetadata, createTag, getConfig } from '../../utils/utils.js';
-// import fetchTaxonomy from '../../scripts/taxonomy.js';
-// import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+import fetchTaxonomy from '../../scripts/taxonomy.js';
+import { updateLinkWithLangRoot } from '../../utils/helpers.js';
+
+function createCategoryLink(doc, categoryEl) {
+  const category = getMetadata('article:tag', doc);
+  return fetchTaxonomy(getConfig(), '/topics')
+    .then((taxonomy) => {
+      const categoryTaxonomy = taxonomy.get(category) || 'News';
+      const categoryLink = createTag('a', { href: updateLinkWithLangRoot(categoryTaxonomy.link) }, categoryTaxonomy.name);
```

Comment:
if `categoryTaxonomy === 'News'` then `categoryTaxonomy.link && categoryTaxonomy.name` will be `undefined`

Reply:
Fixed